frontend/src/pages/news/[newsID].tsx

```diff
@@ -25,37 +19,52 @@ const EachNews = () => {
     padding: '10% 7% 10% 7%',
   };
 
-  const NewsPicStyle = {
+  const NewsPicStyle: CSSProperties = {
     width: '100%',
+    height: '100%',
     border: '1px solid var(--button_border, #34977f)',
+    objectFit: 'cover',
   };
 
-  // TODO : data will be assign to the data got from backend, the newsData will be removed
-  const params = useParams();
-  const data: Props | undefined = newsData.find((news) => news.id.toString() === params.news_id);
-
-  if (data) {
-    return (
-      <div style={{ padding: '10% 10% 0% 10%' }}>
-        <div className='flex_wrapper' style={NewsBgStyle}>
-          <Row>
-            <Col xs={12} md={4} className='center_horizontal'>
-              <img src={data.image_url} style={NewsPicStyle} />
-            </Col>
-            <Col xs={12} md={8}>
-              <h4 className='inpage_title'>{data.title}</h4> <br />
-              <span className='right'>{data.date}</span>
-              <hr className='hr' />
-              <p>{data.subTitle}</p>
-              <p>{data.content}</p>
-            </Col>
-          </Row>
-        </div>
-      </div>
-    );
-  } else {
+  const { status, data } = useQuery({
+    queryKey: ['newsEach'],
```

Comment:
```suggestion
    queryKey: ['newsEach' news_id],
```
make sure there arent same queryKey when fetching "detail" of somwthing like this